src/timeline/js/controllers.js

```diff
@@ -34,9 +34,9 @@ App.controller('TimelineCtrl',function($scope) {
 	$scope.project =
     {
 	  fps: {
-		    num : 24,
-		    den : 1
-		   },
+	    num : 24,
+	    den : 1
+	   },
```

Comment:
Die, tabs, die!

Reply:
(BTW, this still doesn't quite line up, since lines 36-39 start with a single tab and then switch to spaces, while 35 and 40-44 are indented entirely with spaces. But, honestly, this file's tabs-vs-spaces insanity is... clinical, so maybe that's a whole 'nother PR of its own.)

Reply:
@ferdnyc - No worries, I will fix it before merging if possible.